packages/nodes-base/nodes/Postgres/v2/actions/database/insert.operation.ts

```diff
@@ -161,15 +161,28 @@ export async function execute(
 	db: PgpDatabase,
 ): Promise<INodeExecutionData[]> {
 	items = replaceEmptyStringsByNulls(items, nodeOptions.replaceEmptyStrings as boolean);
+	const nodeVersion = nodeOptions.typeVersion as number;
```

Comment:
This code seems to be duplicated a few times. Is there anything we can reduce this code duplication?

Reply:
added helper to remove duplication